contentcuration/contentcuration/frontend/channelList/views/KCard/index.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+
+  <BaseCard
+    :title="title"
+    :headingLevel="headingLevel"
+    :titleLines="titleLines"
+  >
+    <div>
+      <KImg
+        v-if="layout === 'vertical' && thumbnailDisplay !== 'none'"
+        :src="thumbnailSrc"
+        :height="300"
+        :width="500"
+        :isDecorative="false"
```

Comment:
[Required] Thumbnails are decorative images so this needs to be `:isDecorative="true"`. Relatedly, please remove `altText` from one line below because decorative images don't have it.